pvt/helpers/src/math/base.ts

```diff
@@ -1,163 +1,162 @@
-import { bn, decimal, fromFp, toFp } from '../numbers';
+import Decimal from 'decimal.js';
+import { decimal } from '../numbers';
 
-export function computeInvariantMock(fpBalances: bigint[]): bigint {
+export function computeInvariantMock(balances: Decimal[]): Decimal {
```

Comment:
I'd revert these changes.
I think it makes more sense for this functions to work with `bigint`, and just send them bigints as arguments in the tests.

My original comment was more in line with "let's just use `fp` everywhere as opposed to decimals and converting back and forth; e.g.:

```
    const currentBalances = [fp(100), fp(200)];
    const tokenInIndex = 0n;
    const exactBptAmountOut = fp(30);
    const totalSupply = fp(300);
```